cpp/log/frontend.cc

```diff
@@ -24,6 +26,7 @@ Frontend::~Frontend() {
 }
 
 void Frontend::GetStats(Frontend::FrontendStats* stats) const {
+  lock_guard<mutex> lock(stats_mutex_);
```

Comment:
You're a generous man, I would have been tempted to remove the (unused) stats instead... :wink: